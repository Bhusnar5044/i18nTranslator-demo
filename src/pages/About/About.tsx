import { PageContent } from '@components/Layout';
import { FC, memo } from 'react';

export const About: FC = memo(() => {
    return (
        <PageContent>
            <p>This is About Page</p>
        </PageContent>
    );
});

About.displayName = 'About';
