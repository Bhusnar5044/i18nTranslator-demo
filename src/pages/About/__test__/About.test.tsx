import { render } from '@test-utils';
import { About } from '../About';

describe('Home', () => {
    it('should render properly', () => {
        const { container } = render(<About />);
        expect(container).toMatchSnapshot();
    });
});
