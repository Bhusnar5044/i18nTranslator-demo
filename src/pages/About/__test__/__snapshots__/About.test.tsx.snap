// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home > should render properly 1`] = `
<div>
  <main
    class="sc-dujCx eUPcGw"
  >
    <p>
      This is About Page
    </p>
  </main>
</div>
`;

exports[`Home should render properly 1`] = `
<div>
  <main
    class="sc-gQiKMQ jSvYjQ"
  >
    <p>
      This is About Page
    </p>
  </main>
</div>
`;
